techniques/systemSettings/userManagement/userManagement/9.1/userManagement.st

```diff
@@ -392,6 +407,40 @@ bundle agent check_usergroup_user_parameters_&RudderUniqueID&
   methods:
     pass3.showtime::
 
+      ######################
+      # set/check secondary groups
+      # if checkpres, then we are doing dry run only
+      ######################
+      "force_dry_run_mode_${usergroup_user_login[${usergroup_user_index}]}_${usergroup_user_secondary_groupsname[${usergroup_user_index}]}"
```

Comment:
For the 3 promisers below, I am pretty sure we only need to put the logins there

Reply:
no, if we manage the same user in different way, we need both.
index will even be worse if we have 2 directives based on the same technique